dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -0,0 +1,179 @@
+the ingress without having to bother the admins. The router has controls
+to allow the admin to specify whether the users can self-provision
+hostnames, or if they must fit a pattern the admin defines. The other
+solutions require the admin to do it, or it requires that they delegate
+a lot of privilege.
+
+A set of routes can be created in the various projects. The overall
+set of routes is available to the set of routers. Each router selects
+from the set of routes.  All routers see all routes unless restricted
+by labels on the router, this is called router
+xref:../architecture/core_concepts/routes.adoc#router-sharding[sharding].
+
+The xref:../admin_guide/high_availability.adoc#admin-guide-high-availability[High Availability]
+section describes how to configure a router for high availability service
+using multiple replicas.
+
+[[using-the-loadbalancer]]
+== Using a Service of Type LoadBalancer
+
+There are load balancers available on
+xref:../install_config/configuring_aws.adoc#install-config-configuring-aws[AWS] and
+xref:../install_config/configuring_gce.adoc#install-config-configuring-gce[GCE] clouds as well as
+xref:../admin_guide/tcp_ingress_external_ports.adoc#admin-guide-expose-external-ports[non-cloud].
+
+The non-cloud
+xref:../admin_guide/tcp_ingress_external_ports.adoc#admin-guide-expose-external-ports[LoadBalancer] allocates
+a unique IP from a configured pool. This limits you to a single ingress IP
+(can be a VIP, but still will be a single machine for initial load balancing).
+It simplifies the admin job by providing the needed IP address, but uses one
+IP per service.
+
+Load balancing is described
+link:http://kubernetes.io/docs/user-guide/services/#type-loadbalancer[here].
+
+[[using-externalIP]]
+== Using a Service ExternalIP
+
+The user can manually assign
+link:http://kubernetes.io/docs/user-guide/services/#external-ips[ExternalIPs]
+to the service. The supplied list of IP addresses are used for load balancing
+incoming requests. The service port is opened on all nodes running kube-proxy.
+Doing this requires elevated permissions to assign, and manual tracking of the
+IP:ports that are in use.
+
+An externally visible IP can be configured in one of several ways:
+
+- manually configure the ExternalIP with a known external IP address.
+- configure ExternalIP to a
+xref:../admin_guide/high_availability.adoc#admin-guide-high-availability[VIP]
+that is generated from (VRRP)
+- In a cloud (xref:../install_config/configuring_aws.adoc#install-config-configuring-aws[AWS] or
+xref:../install_config/configuring_gce.adoc#install-config-configuring-gce[GCE])
+use type=LoadBalancer
+- In a non-cloud environment, configure an ingressIP range (IngressIPNetworkCIDR),
+service.type=LoadBalancer and service.port.ingressIP
+
+The admin must make sure the external IPs are routed to the nodes and local
+firewall rules on all nodes allow access to the open port.
+
+Use the same nodes as the router but advertise them with VRRP and use
```

Comment:
Consider highlighting the utility of this action.  At present its relation to the rest of the section is not clear.